### PR TITLE
Add X++

### PR DESCRIPTION
**Pull request by devikabhapkar**
## Adding a language

- [x] The code displays "Hello World" ([tio.run](https://tio.run) may help for testing)
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme
- [x] I have no association with the language
- [x] There are no copyright issues with this code
- [x] The language has not been added prior to this pull request

#### Link to programming language: 
https://learn.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/dev-ref/xpp-language-reference

### CODE REVIEW

**Comment by devikabhapkar** [comment]
Hi @defunkt @henrik @robbyrussell @foca 
Can you please check my submission?

Looking forward to hearing from you!!

File: x/X++.xpp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class classHello
+{
+}
+
+static void main(args Args)
+{
+ dialog dialog;
+ dialog = new dialog();
+ dialog.addText("Hello World!");
```
Comment (MrBrain295) [inline x/X++.xpp:9]:
```suggestion
 dialog.addText("Hello World");
```

File: x/X++.xpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class classHello
```
Comment (Richienb) [inline x/X++.xpp:1]: Is this class necessary?

File: x/X++.xpp [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+{
+}
+
+static void main(args Args)
+{
+ dialog dialog;
+ dialog = new dialog();
```
Comment (Richienb) [inline x/X++.xpp:8]: Is it possible for these two lines to be joined together?

File: x/X++.xpp (hunk 2 above)
Comment (devikabhapkar) [reply x/X++.xpp:1]: Yes class is necessary

File: x/X++.xpp (hunk 3 above)
Comment (devikabhapkar) [reply x/X++.xpp:8]: No

**Comment by devikabhapkar** [comment]
Hi @Richienb 
I have updated changes.

**Pull request merged**